### PR TITLE
Staging test

### PR DESCRIPTION
**Pull request by bago2k4**
This is a a one monholitic PR that contains all these PRs:
https://github.com/open-company/open-company-web/pull/188
https://github.com/open-company/open-company-web/pull/193
https://github.com/open-company/open-company-web/pull/194
and this cards:
https://trello.com/c/IJmEJTNx
https://trello.com/c/Lui7NoNj

check the various link for testing

### CODE REVIEW

**Comment by bago2k4** [comment]
[More robust client-side error handling](https://trello.com/c/Lui7NoNj/683-more-robust-client-side-error-handling)

**Comment by belucid** [comment]
Reviewing.

**Comment by belucid** [comment]
@bago2k4 

Do we use these? What do we use these for?

```
resources/public/css/images/overlay.png
resources/public/css/images/overlay1.jpg
resources/public/css/images/overlay2.jpg
resources/public/css/images/overlay3.jpg
```

**Comment by bago2k4** [comment]
@belucid looking at tests now.
i really don't know what those overlay images are for, they looks like the old navbar that we were using for the static pages but i changed the navbar now.. should i remove those?
a grep on the whole project doesn't give anything looking for `overlay.png` or `overlay1.jpg` etc..

**Comment by belucid** [comment]
@bago2k4 yeah, I looked through the project and couldn't find that we used them, that's why I asked. Yes. Let's remove them.

**Comment by belucid** [comment]
@bago2k4 there's also `xoverlay.png`

**Comment by belucid** [comment]
@bago2k4 and some images from the old web design.

I'll remove all these. No worries!

File: src/open_company_web/components/topic_list.cljs [hunk 1]
```diff
@@ -46,7 +46,7 @@
         new-topics (concat old-topics [new-topic])
         new-topic-kw (keyword new-topic)]
     (if section-data
-      (dispatcher/dispatch! [:start-foce new-topic-kw (or section-data {:section new-topic :placeholder true})])
+      (dispatcher/dispatch! [:start-foce new-topic-kw (or section-data {:section new-topic :placeholder true :body-placeholder "What would you like to say about this?"})])
```
Comment (belucid) [inline src/open_company_web/components/topic_list.cljs]: Why are we passing in this body placeholder here?

**Comment by belucid** [comment]
@bago2k4 

- [] the footer on the new look 404/500 pages is not at the bottom of the screen on long displays.

File: scss/public/css/site.main.scss [hunk 2]
```diff
@@ -52,7 +53,7 @@ h3 {
 /*--- Nav / Header ---*/
 
 div.gradient {
-  background-image: linear-gradient(138deg, #4183D7 0%, #51A0CB 44%, #54A6C9 51%, #56ABC6 59%, #65C6BB 100%);
+  background-image: linear-gradient(138deg, #f9d748 0%, #51A0CB 44%, #54A6C9 51%, #56ABC6 59%, #65C6BB 100%);
```
Comment (bago2k4) [inline scss/public/css/site.main.scss]: done

File: site/oc/core.clj [hunk 3]
```diff
@@ -68,16 +86,26 @@
...
+  [:nav.navbar.navbar-default.navbar-bottom
+    [:ul.nav.navbar-nav.navbar-left.navbar-bottom-left
+      [:li [:a.navbar-logo {:href "/"}
+        [:img {:alt "OpenCompany" :src "img/oc-logo-grey.svg"}]]]
+      [:li.web-only
+        [:a {:href "/pricing"} "Pricing"]]
+      [:li.web-only
+        [:a {:href "/about"} "About"]]
+      [:li
+        [:a.contact {:href "mailto:hello@opencompany.com"} "Contact"]]
```
Comment (bago2k4) [inline site/oc/core.clj]: same as above

File: site/oc/pages.clj [hunk 4]
```diff
@@ -148,6 +148,22 @@
...
+(defn server-error [{contact-mail-to :contact-mail-to contact-email :contact-email}]
+  [:div.container.outer.section.content.about
+   [:div.container.inner
+    [:div.row
+     [:div.col-md-12
+      [:div.not-found
+       [:h1 "500"]
+       [:h2 "Hmm, this does not look right."]
+       [:p
+        "You seem to have come accross an error."
```
Comment (bago2k4) [inline site/oc/pages.clj]: done

**Pull request merged**